src/main/java/org/apache/commons/fileupload/FileUploadBase.java

```diff
@@ -138,6 +138,11 @@ public static boolean isMultipartContent(final HttpServletRequest req) {
      */
     public static final String MULTIPART_MIXED = "multipart/mixed";
 
+    /**
+     * HTTP content type header for multiple related data.
+     */
+    public static final String MULTIPART_RELATED = "multipart/related";
```

Comment:
New public and protected items need a Javadoc 'since tag.

Reply:
I've added a `@since 1.6` comment. Hope this is correct